server/utils/placementRequests/table.ts

```diff
@@ -35,7 +35,22 @@ export const tableRows = (tasks: Array<PlacementRequestTask>): Array<TableRow> =
 export const dashboardTableRows = (
   placementRequests: Array<PlacementRequest>,
   status: PlacementRequestStatus | undefined,
+  showRequestedAndActualArrivalDates: boolean = false,
```

Comment:
could we have this third param as an object? 
I know it should be short lived but I find it a lot clearer to read in the consuming code than just passing a boolean